script.adzapper/service.py

```diff
@@ -0,0 +1,298 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import urllib
+import traceback
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcvfs
+
+if sys.version_info >= (2, 7):
+    import json
+else:
+    import simplejson as json
+
+# Import the common settings
+from resources.lib.settings import log
+from resources.lib.settings import Settings
+from resources.lib.timer import TimerWindow
+
+ADDON = xbmcaddon.Addon(id='script.adzapper')
+
+def jsonrpc(query):
+    querystring = {"jsonrpc": "2.0", "id": 1}
+    querystring.update(query)
+    try:
+        response = json.loads(xbmc.executeJSONRPC(json.dumps(querystring, encoding='utf-8')))
+        return response
+    except TypeError, e:
```

Comment:
`except TypeError as e` for python3 compatibility

Reply:
can you please give me the new except line how it should be.
Not so much python 3 knowledge here.

Reply:
sorry got it, updated PR comes